website/docs/src/content/cloud/observability.mdx

```diff
@@ -0,0 +1,269 @@
+
+```tsx
+// Executing a workflow with trace URL printing
+const result = await MyWorkflow.run(
+  { input: "What is GenSX?" },
+  { printUrl: true },
+);
+
+// Console output includes:
+// [GenSX] View execution at: https://app.gensx.com/your_org/executions/your_execution_idf
+```
+
+### Viewing the flame graph
+
+The flame graph visualizes the entire execution tree including branches, all nested sub-components, and timing:
+
+![Workflow component tree](/cloud/trace.png)
+
+- **Component hierarchy**: See the nested structure of your components and their relationships
+- **Execution timing**: The width of each bar represents the relative execution time
+- **Status indicators**: Quickly spot errors or warnings with color coding
+- **Component filtering**: Focus on specific components or component types
+
+Click on any component in the flame graph to inspect its details, including inputs, outputs, and timing information.
+
+### Viewing component inputs and outputs
+
+For each component in your workflow, you can inspect:
+
+1. **Input properties**: All props passed to the component
+2. **Output values**: The data returned by the component
+3. **Execution duration**: How long the component took to execute
+4. **Metadata**: Additional information like token counts for LLM calls
+
+![Component trace](/cloud/component-trace.png)
+
+This visualization is particularly valuable for debugging production and user-reported issues like hallucinations.
+
+### Viewing historical traces
+
+The GenSX Console maintains a history of all your workflow executions, allowing you to:
+
+- **Compare executions**: See how behavior changes across different runs
+- **Identify patterns**: Spot recurring issues or performance bottlenecks
+- **Filter by status**: Focus on successful, failed, or in-progress executions
+- **Search**: Find historical executions
+
+Historical traces are automatically organized by project and environment, making it easy to find relevant executions.
+
+## Configuring traces
+
+tGenSX provides flexible options for configuring and organizing traces for the GenSX Cloud serverless platform, local development, and any other deployment platform like vercel, cloudflare and AWS.
+
+### Tracing GenSX Cloud workflows
+
+When running workflows on GenSX Cloud, tracing is automatically configured:
+
+- **Project context**: Traces are associated with the correct project
+- **Environment segregation**: Development, staging, and production traces are kept separate
+- **Authentication**: API keys and organization information are handled automatically
+- **Retention**: Traces are stored according to your plan limits
+
+No additional configuration is needed – everything works out of the box.
+
+### Tracing on other deployment platforms
+
+To enable tracing for workflows deployed outside of GenSX Cloud (like AWS Lambda, Google Cloud Functions, etc.), you need to set several environment variables:
```

Comment:
Fixed the name of GCP Cloud Run (if it's too long "Azure Functions" would be shorter :))

```suggestion
To enable tracing for workflows deployed outside of GenSX Cloud (like AWS Lambda, GCP Cloud Run functions, etc.), you need to set several environment variables:
```